nntrainer/layers/tflite_layer.cpp

```diff
@@ -126,7 +126,7 @@ void TfLiteLayer::forwarding(sharedConstTensors in) {
   if (status != kTfLiteOk)
     throw std::runtime_error("Invoke failed");
 
-  net_hidden[0]->var = *out[0];
+  net_hidden[0]->getVariableRef() = *out[0];
```

Comment:
I guess we can merge this with getVariable() as a good first issue later

Reply:
you mean make getVariableRef() and getVariable() to be the same?

Reply:
Yes. Do we need to differentiate those two by any chance?

Reply:
will add as a issue for later.

Reply:
Alright :)